src/sdk/resources/transferSecurityMappings.ts

```diff
@@ -0,0 +1,15 @@
+import { IHttpClient } from '../http';
+import { getPagedResponse, getTypedResponse, Resource, PagedResult } from './resource';
+import { TransferSecurityMapping } from '../models';
+
+export class TransferSecurityMappingsResource extends Resource<TransferSecurityMapping> {
+	constructor(httpClient: IHttpClient) {
+		super(httpClient, 'item', 'item', null, null, 'transfers');
+	}
+
+	async page(params?: any): Promise<PagedResult<TransferSecurityMapping>> {
+		const result = await this.httpClient.get(`${this.resourcePath}/${params && params.id}/security_map`, this.mergeDefaultParams(params));
```

Comment:
This feels wrong but I suppose we can worry about fixing it once the services have support for exporting security maps for more than one job.